app/controllers/application_controler.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
```

Comment:
- [ ] Your application controller files are misnamed; hence the app is not running correctly.  Kindly rename this file to `application_controller` instead of `application_controler`. 

![image](https://user-images.githubusercontent.com/12745474/206322052-950f0bb7-18f9-46c7-9492-938095b2010a.png)
![image](https://user-images.githubusercontent.com/12745474/206322157-a5de3e71-1d8c-46d3-b7b8-649570ef3230.png)

Reply:
Done :heavy_check_mark: